src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/servicebus/CreateUpdatesBusSender.java

```diff
@@ -1,44 +1,54 @@
 package uk.gov.hmcts.ethos.replacement.docmosis.servicebus;
```

Comment:
I think you need to change the names in this class / package and also in the methods as well to reflect what its doing now.

Reply:
Since we're still supporting both mechanisms depending on feature flag I was thinking a refactor pass could be done when the servicebus is removed entirely to rename this